content/push/index.textile

```diff
@@ -78,4 +78,10 @@ Each push target device is associated with a unique @deviceId@ and authenticatio
 
 * By employing a standard "Ably key":/auth/basic or "Token":/auth/token, with a @deviceIdentityToken@, a credential generated during registration to assert the device's identity, included in the request header.
 
-The sevice credential management is handled by the "Ably SDK":https://ably.com/docs/sdks, removing the need for the client application to manage device credentials unless accessing the "push admin API":/api/realtime-sdk/push-admin directly via HTTP.
\ No newline at end of file
+The sevice credential management is handled by the "Ably SDK":https://ably.com/docs/sdks, removing the need for the client application to manage device credentials unless accessing the "push admin API":/api/realtime-sdk/push-admin directly via HTTP.
+
+h4(#Error). Error Handling
+
+The "metachannel":/metadata-stats/metadata/subscribe#metachannels @[meta]log:push@ handles errors in the push notification system, providing a dedicated stream for logging errors such as delivery failures, authentication issues, and configuration problems. 
+
+When subscribed to @[meta]log:push@ you receive realtime error updates, complete with error codes, messages, and timestamps. This information helps diagnose and resolve issues promptly.
```

Comment:
Mmm, from testing previously I'm pretty sure all of this info is wrapped within a standard message body. Ideally we should have an example payload.